CMakeLists.txt

```diff
@@ -278,6 +278,37 @@ if(LOG_CONSUMER_DEFAULT STREQUAL "STDOUTERR")
     set(STDOUTERR_LOG_CONSUMER ON)
 endif()
 
+###############################################################################
+# Log Info default setup
+###############################################################################
+if(CMAKE_BUILD_TYPE STREQUAL "Debug")
+    option(LOG_NO_INFO "Do not compile Info Log level" OFF)
```

Comment:
I do not see difference in the behaviour.
I think it is better this way as the defaul value is different depending the *build type*.
The other way around would suggest that it has a default value that is changed in a specific *build type* (almost the same, but I still think the first interpretation describes better the behaviour).

Reply:
How about [CMakeDependentOption](https://cmake.org/cmake/help/latest/module/CMakeDependentOption.html)

Reply:
Forget it, that would not make the option available when the condition is false

Reply:
Yes, but you are giving different descriptions to the same CMake option. What I mean is that the option description must be unique since the option means only one thing, not two. It is the behavior what is different depending on the value of the option.